### PR TITLE
Update to the new logstah-core-plugin-api

### PR DESCRIPTION
**Pull request by ph**
Ref: https://github.com/elastic/logstash/issues/5269

### CODE REVIEW

File: logstash-output-example.gemspec [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
   s.name = 'logstash-output-example'
-  s.version         = "2.0.0"
-  s.licenses = ["Apache License (2.0)"]
+  s.version = '1.0.0'
```
Comment (purbon) [inline logstash-output-example.gemspec:3]: downgrade version?

Comment (purbon) [inline logstash-output-example.gemspec:3]: we should have might be all the same or have different paths, isn't?

**Comment by purbon** [comment]
also from tests:

```
LoadError: no such file to load -- logstash/codecs/plain
          require at org/jruby/RubyKernel.java:1040
          require at /home/travis/build/logstash-plugins/logstash-output-example/vendor/bundle/jruby/1.9/gems/polyglot-0.3.5/lib/polyglot.rb:65
           (root) at /home/travis/build/logstash-plugins/logstash-output-example/spec/outputs/example_spec.rb:4
             load at org/jruby/RubyKernel.java:1059
           (root) at /home/travis/build/logstash-plugins/logstash-output-example/vendor/bundle/jruby/1.9/gems/rspec-core-3.4.4/lib/rspec/core/configuration.rb:1
```

**Comment by purbon** [comment]
```
LoadError: JRuby ext built for wrong Java version in `JrubyEventExtService': java.lang.UnsupportedClassVersionError: JrubyEventExtService : Unsupported major.minor version 52.0
  require at org/jruby/RubyKernel.java:1040
   (root) at /home/travis/build/logstash-plugins/logstash-output-example/vendor/bundle/jruby/1.9/gems/logstash-core-event-java-5.0.0.alpha5.snapshot1-java/lib/logstash-core-event-java/logstash-core-event-java.rb:25
  require at org/jruby/RubyKernel.java:1040
   (root) at /home/travis/build/logstash-plugins/logstash-output-example/vendor/bundle/jruby/1.9/gems/logstash-core-event-java-5.0.0.alpha5.snapshot1-java/lib/logstash-core-event.rb:1
  require at org/jruby/RubyKernel.java:1040
   (root) at /home/travis/build/logstash-plugins
```

wrong java version for travis?

**Comment by ph** [comment]
@purbon travis is now green.

**Pull request merged**